features/offer/offer-template-creation.feature

```diff
@@ -6,36 +6,36 @@ Feature: Create an offer template
 
   Scenario Outline: offer template is created
     Given the server is up
-    When an offer template is created in the "offer-template-game" game with:
```

Comment:
That way it does not get the values from the table. It adds in the payload the strings <name>, <pid>, <contents>, etc., instead of oc1, com.tfg.sample. {'x': 1}, etc.